kolibri/core/assets/src/views/Navbar/NavbarLink.vue

```diff
@@ -57,11 +57,15 @@
         required: true,
         validator: validateLinkObject,
       },
+      textColor: {
```

Comment:
Could we avoid prop drilling here, and just reference the value directly? NavbarLink is _only_ used in the Navbar component, and we remove it from the public API completely in develop, so we can just set it directly to the value we want.

Reply:
Good catch - just updated this